cliconfig/loader.go

```diff
@@ -225,7 +225,11 @@ func (l Loader) setFieldValueFromCLI(fieldName string, cliName string) error {
 				case reflect.String:
 					value = configFileValue
 				case reflect.Slice:
-					value = strings.Split(configFileValue, ",")
+					valueSlice := strings.Split(configFileValue, ",")
```

Comment:
Good move, it seems very unlikely that any string list will be dependent on whitespace.